README.md

```diff
@@ -0,0 +1,38 @@
+# Testplane Docs Website
+
+Testplane is a modern end-to-end testing framework, featuring support for every browser with a single API, testing on real devices, great extensibility and vast visual testing capabilities.
```

Comment:
imo, this sentence is hard to read:

> featuring support for every browser with a single API, testing on real devices, great extensibility and vast visual testing capabilities

after `with a single API`, it takes a while to get `testing on real devices...` actually refers to `featuring`...

Maybe turn this into list, or nobody usually does that in the section?

Reply:
Split into 2 sentences: `Testplane is a modern and robust end-to-end testing framework. It features support for every browser with a single API, testing on real devices, great extensibility and vast visual testing capabilities.`

Don't want to have a list here, because it's meant to be a short description of what a tool is.